ProjectSidewalk/SidewalkWebpage
# 2771: Adding popups to comment/feedback submissions

Resolves #2771 

Originally there was no visual feedback from the website when submitting a comment on /labelMap and feedback on /audit and /validate that allowed users to know that their comments and feedback were being submitted. For /labelMap, I have added a "comment submitted" popup that shows for about 2s above the submission button and have disabled the button for that duration whenever a comment is successfully submitted. For /audit and /validate, a popup saying "feedback submitted" appears over the feedback button once the comment is successfully submitted. 

Before:
*No visuals to show

After:
Label Map:
![image](https://user-images.githubusercontent.com/61511682/176350327-5dda8b73-0a54-4198-bd24-96ebb3967db1.png)

Audit:
![image](https://user-images.githubusercontent.com/61511682/176350347-0f54785a-6c3a-405e-9275-4ae8bda23afc.png)

Validate:
![image](https://user-images.githubusercontent.com/61511682/176350360-d74e8212-cedf-4daa-a63d-79c285f2e8ba.png)

Testing instructions (do on all three pages):
1. (Chrome can do this, but not sure about others) Open the developer tools on the side, so that you can see if the console displays any errors
2. Type in something random into the feedback/comment textfield
3. Submit (You should see the popup)
4. If on /labelMap, try clicking the submission button during the time that the popup is up to see if it toggles (it should not)
